src/main/kotlin/org/objectionary/ddr/launch/Combiner.kt

```diff
@@ -48,11 +48,11 @@ val documents: MutableMap<Document, String> = mutableMapOf()
  * Aggregates all steps of analysis
  *
  * @param path path to the directory to be analysed
- * @param dirName postfix of the resulting directory
+ * @param dirPostfix postfix of the resulting directory
  */
-fun launch(path: String, dirName: String = "ddr") {
+fun launch(path: String, dirPostfix: String = "ddr") {
```

Comment:
@c71n93 It's better to pick simple nouns for variables. If a variable has compound name it most probably has too wide visibility scope. In this case: can `dirPostfix` be renamed into just `postfix`?

Reply:
@mximp I agree, the `postfix` name will be enough here.